### PR TITLE
[dem] Concentric element size coarsener

### PR DESCRIPTION
**Pull request by maceligueta**
From a monodisperse mesh, this script coarsens the mesh according to a size law.

### CODE REVIEW

File: applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    distance_to_origin = math.sqrt(x*x + y*y)
+    max_distance_for_fine_radius = 0.00762
+    slope = 0.01
+    if distance_to_origin < max_distance_for_fine_radius:
+        radius = fine_radius
+    else:
+        radius = fine_radius + slope * (distance_to_origin - max_distance_for_fine_radius)
+
+    #if x>0.003918 and x<0.00392 and y>-9.724e-05 and y<-9.72e-05:
+    #    print(radius)
```
Comment (salvalatorre) [inline applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py]: Do you want to keep this?

File: applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        self._GetDeviationFromMeanSizeOfAllParticles()
+        for props in self.spheres_model_part.Properties:
+            self.list_of_frictions_at_start.append(props[DEM.FRICTION])
+            props.SetValue(DEM.FRICTION, self.size_modifier_parameters["material_settings"]["friction"].GetDouble())
+            self.list_of_young_modulus_at_start.append(props[KratosMultiphysics.YOUNG_MODULUS])
+            props.SetValue(KratosMultiphysics.YOUNG_MODULUS, self.size_modifier_parameters["material_settings"]["young_modulus"].GetDouble())
+            self.list_of_coefficients_of_restittution_at_start.append(props[DEM.COEFFICIENT_OF_RESTITUTION])
+            props.SetValue(DEM.COEFFICIENT_OF_RESTITUTION, self.size_modifier_parameters["material_settings"]["coefficient_of_restitution"].GetDouble())
+            #self.list_of_damping_gammas_at_start.append(props[DEM.DAMPING_GAMMA])
+            #props.SetValue(DEM.DAMPING_GAMMA, self.size_modifier_parameters["material_settings"]["young_modulus"].GetDouble())
```
Comment (salvalatorre) [inline applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py]: And this?

File: applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py (hunk 1 above)
Comment (maceligueta) [reply applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py]: Thanks!

File: applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py (hunk 2 above)
Comment (maceligueta) [reply applications/DEMApplication/python_scripts/concentric_element_size_coarsener.py]: No, thanks!

**Review by salvalatorre — APPROVED** [review]
Good job!

**Pull request merged**